src/handlers/payout/post.ts

```diff
@@ -1,9 +1,11 @@
 import { getWalletAddress } from "../../adapters/supabase";
 import { getBotConfig, getBotContext, getLogger } from "../../bindings";
 import { addCommentToIssue, generatePermit2Signature, getAllIssueComments, getIssueDescription, getTokenSymbol, parseComments } from "../../helpers";
-import { MarkdownItem, Payload, UserType, CommentElementPricing } from "../../types";
+import { Incentives, Payload, UserType } from "../../types";
+import { commentParser } from "../comment";
+import Decimal from "decimal.js";
 
-const ItemsToExclude: string[] = [MarkdownItem.BlockQuote];
+const ItemsToExclude: string[] = ["blockquote"];
```

Comment:
I feel like excludes should also be configurable. Perhaps as part of another task.

Reply:
Setting `blockquote` [pricing](https://github.com/ubiquity/ubiquibot/blob/2e31b3f545cbf7314421d7f7e0c6fc8ea8a140e8/ubiquibot-config-default.json#L97) to 0 should have the same effect

Reply:
Actually it has a different effect. When a tag is in excluded items, the parser won't go recursively inside it. In practice it means the whole blockquote is omitted. 

If you only set `blockquote: 0` in the config then the parser will still count other tags inside the blockquote.

Reply:
> Setting `blockquote` [pricing](https://github.com/ubiquity/ubiquibot/blob/2e31b3f545cbf7314421d7f7e0c6fc8ea8a140e8/ubiquibot-config-default.json#L97) to 0 should have the same effect

Maybe we can set to -0.1 or whatever the word count credit is lol

Reply:
> > Setting `blockquote` [pricing](https://github.com/ubiquity/ubiquibot/blob/2e31b3f545cbf7314421d7f7e0c6fc8ea8a140e8/ubiquibot-config-default.json#L97) to 0 should have the same effect
> 
> Maybe we can set to `-0.1` or whatever the generic word count incentive is lol

I think it is better to explicitly exclude such tags from recursive parsing (as stated [here](https://github.com/ubiquity/ubiquibot/issues/612))

Reply:
> > > Setting `blockquote` [pricing](https://github.com/ubiquity/ubiquibot/blob/2e31b3f545cbf7314421d7f7e0c6fc8ea8a140e8/ubiquibot-config-default.json#L97) to 0 should have the same effect
> > 
> > 
> > Maybe we can set to `-0.1` or whatever the generic word count incentive is lol
> 
> I think it is better to explicitly exclude such tags from recursive parsing (as stated [here](https://github.com/ubiquity/ubiquibot/issues/612))

- Updated https://github.com/ubiquity/ubiquibot/issues/772